style.css

```diff
@@ -508,4 +508,63 @@ main > div img {
  div.evidon-notice-link {
    display:none;
  }
- 
\ No newline at end of file
+ 
+  /*
+  * Styles for Dropdown Region Selector
+  */
+.region-dropdown {
+  border: 1px solid #d3d3d3;
+  background-color: #ffffff;
+  border-radius: 4px;
+  display: inline-block;
+  font-size: inherit;
+  padding: 3px 15px 5px 8px;
+  margin: 0;
+  visibility: hidden;
+  position: absolute;
+  z-index: 0;
+}
+
+.region-dropdown.visible {
+  visibility: visible;
+  z-index: 2;
```

Comment:
Why do you also need to change `z-index`?

Reply:
if in case not using z-index: 2 and having a different footer Feds experience
![Screen Shot 2021-02-18 at 11 21 31 AM](https://user-images.githubusercontent.com/62952234/108343074-d0cffc00-71db-11eb-978b-b47a90ad4c99.png)

with z-index: 2
![Screen Shot 2021-02-18 at 11 21 40 AM](https://user-images.githubusercontent.com/62952234/108343190-f2c97e80-71db-11eb-8831-73e18a61e57e.png)